exporters/zipkin/src/test/java/io/opentelemetry/exporters/zipkin/ZipkinSpanExporterTest.java

```diff
@@ -236,7 +238,7 @@ public void testExport_failed() throws IOException {
     when(mockZipkinCall.execute()).thenThrow(new IOException());
 
     ResultCode resultCode =
-        zipkinSpanExporter.export(Collections.singleton(buildStandardSpan().build()));
+        zipkinSpanExporter.export(Collections.<SpanData>singleton(buildStandardSpan().build()));
```

Comment:
Use singletonList here?

Reply:
better than a single-item set? Since it takes Collection, either should work, correct?

Reply:
Correct just that list seems lightweight compared to set :)

Reply:
if you look at the implementation of `singleton`, I think it's about as lightweight as it's possible to be. :)

Reply:
implementation-wise, they're really just a wrapper for a single data item. I like using `singleton` as it's 4 less characters than `singletonList`. But, honestly, I don't care all that much, either way.

Reply:
Leave it as it is, for me was just reading the code and found this inconsistencies, so was more like this is not what I was expecting.